lib/Service/src/Util.h

```diff
@@ -85,35 +86,61 @@ namespace Util
 
     /**
      * Divide and round.
-     * 
+     *
      * @param[in] numerator     The numerator.
      * @param[in] denominator   The denominator.
-     * 
+     *
      * @return Result
      */
     uint32_t divRoundUp(uint32_t numerator, uint32_t denominator);
 
     /**
      * Divide and round.
-     * 
+     *
      * @param[in] numerator     The numerator.
      * @param[in] denominator   The denominator.
-     * 
+     *
      * @return Result
      */
     int32_t divRoundUp(int32_t numerator, int32_t denominator);
 
     /**
      * Is button triggered?
      * Triggered means a pressed/released change.
-     * 
+     *
      * @param[in]       button      The button.
      * @param[inout]    lastState   The last button state.
-     * 
+     *
      * @return If button is triggered, it will return true otherwise false.
      */
     bool isButtonTriggered(IButton& button, bool& lastState);
 
+    /**
+     * Convert a speed in encoder steps per second to a speed in mm/s.
+     *
+     * @param[in]   speedStepsPerSec Speed in encoder steps per second
```

Comment:
Sorry for being nitpicking - missing dots at the of lines (same in Doxygen comment of `millimetersPerSecondToStepsPerSecond`)